backend/src/kitconcept/intranet/profiles/default/types/Document.xml

```diff
@@ -4,13 +4,16 @@
         name="Document"
         i18n:domain="plone"
 >
+
   <property name="behaviors"
             purge="false"
   >
 
     <element value="kitconcept.intranet.location" />
     <element value="kitconcept.intranet.organisational_unit" />
     <element value="kitconcept.intranet.lcm" />
+    <element value="kitconcept.intranet.votes" />
+    <element value="kitconcept.intranet.enable_likes" />
```

Comment:
@iFlameing Timo wants a global setting to turn content rating on/off (see: https://gitlab.kitconcept.io/kitconcept/distribution-kitconcept-intranet/-/issues/56#note_21902). This is important because we want it on for new sites, but off for upgraded sites like BRH (until an admin decides to turn it on).

You can add the setting and control panel, then I can figure out how to prevent the "enable likes" field from showing up when it is disabled globally.